liblangutil/SourceReferenceFormatterHuman.cpp

```diff
@@ -70,7 +70,7 @@ void SourceReferenceFormatterHuman::printSourceLocation(SourceReference const& _
 	if (_ref.sourceName.empty())
 		return; // Nothing we can print here
 
-	int const leftpad = static_cast<int>(log10(max(_ref.position.line, 1))) + 1;
```

Comment:
Could we make this more obviously correct by using
```
string lineNr = to_string(_ref.position.line);
int const leftpad = lineNr.size();
```
and actually using the lineNr string below instead of `_ref.position.line`?
And maybe even replace `leftpad` by `lineNr.size()`?

Reply:
if 10, 100, 1000....0 is the problem, why didn't you just `log10(max(_ref.position.line + 1, 1)) + 1` instead? (the added ` + 1` in the log10? Shouldn't that do the fix? @chriseth @a3d4

Reply:
@christianparpart 
> why didn't you just log10(max(_ref.position.line + 1, 1)) + 1 instead?

I did, it was my initial proposal, see 3faaab7e9 :).

There is one issue with `log10` though. Strictly speaking, we cannot guarantee that `log(100)` is `2.0`, and not something like `1.99...9`.